### PR TITLE
.gitignore: Add .pytest_cache

### LINKED ISSUE #536: .gitignore: Add .pytest_cache
difficulty/newcomer

### PR DESCRIPTION
**Pull request by shikharvaish28**
Closes #536

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,6 +45,8 @@ nosetests.xml
 coverage.xml
 *,cover
 .hypothesis/
+.pytest_cache/
+
```
Comment (MacBox7) [inline .gitignore]: Drop this extra line.

**Comment by jayvdb** [comment]
ack https://github.com/coala/corobo/pull/566/commits/cd03e3ae2de48ce5647c5c862e42891af04ec31c

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by shikharvaish28** [comment]
[coala/corobo](https://trello.com/c/17wqioRI/5-coala-corobo)

**Pull request merged**